pkg/components/imetrics/iprom.go

```diff
@@ -79,6 +80,10 @@ func NewPrometheusReporter(cfg *PrometheusConfig, manager *connector.PrometheusM
 			Name: attr.VendorPrefix + "_instrumentation_errors_total",
 			Help: "Total number of instrumentation errors by process and error type",
 		}, []string{"process_name", "error_type"}),
+		avoidedServices: prometheus.NewGaugeVec(prometheus.GaugeOpts{
+			Name: attr.VendorPrefix + "_avoided_services",
+			Help: "Services avoided by Beyla due to existing OpenTelemetry instrumentation",
```

Comment:
Mention of Beyla in the description. Let's avoid the product name or use the vendor prefix if needed.